### PR TITLE
[WIP] PHP 7.0 support

### PR DESCRIPTION
**Pull request by chirimoya**
__tasks:__

- [ ] test coverage

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes https://github.com/sulu-io/sulu-standard/issues/583
| Related PRs      | https://github.com/sulu-io/sulu-standard/pull/579
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ sudo: false
 language: php
 
 php:
-  - 5.5
+  - 7.0
```
Comment (danrot) [inline .travis.yml]: Isn't that a bit risky? I''d say we should at least also run against 5.6.

Comment (chirimoya) [inline .travis.yml]: only until tests are not failing any more with php 7.0

**Comment by chirimoya** [comment]
@dantleech Since we update `symfony-cmf/testing` to `dev-master` the memory consumption increased and now tests are failing for php5.

https://travis-ci.org/sulu-io/sulu/jobs/98287654#L620

https://blackfire.io/profiles/efa4c86a-7378-4db2-b635-560ff1bd2acd/graph
https://blackfire.io/profiles/7f4393f1-f512-4c3c-a566-c68252a9c7c4/graph

If you have a look at the profile-metrics you will notice that for the "good" one the garbage collector is called 25 times and for the "poor" one none.

Any idea?

/cc @lsmith77  @dbu

**Comment by wachterjohannes** [comment]
i reactivate the garbage-collector in the `SuluTestCase` to avoid memory issues. it was deactivated by https://github.com/symfony/symfony/pull/13398/files#diff-81bfee6017752d99d3119f4ddb1a09edR13

**Pull request merged**